src/index.js

```diff
@@ -71,158 +71,188 @@ class App extends React.Component {
     if (file.type === 'application/json') {
       const fr = new FileReader();
 
-      fr.onload = (evt) => {
+      fr.onload = evt => {
```

Comment:
why evt? consistency with the rest of the file please 🙏

Reply:
good catch, it was from a copy and paste 🤦‍♂️